pages/services/edge-lb/1.4/cloud-connector/index.md

```diff
@@ -0,0 +1,652 @@
+
+1. Open a text editor, then copy and paste the following sample app definition to create the `host-echo.json` file:
+
+  ```json
+  {
+      "id": "/host-echo",
+      "cmd": "/start $PORT0",
+      "instances": 1,
+      "cpus": 0.1,
+      "mem": 32,
+      "constraints": [["public_ip", "UNLIKE", "true"]],
+      "container": {
+          "type": "DOCKER",
+          "docker": {
+              "image": "mesosphere/echo-http"
+          }
+      },
+      "portDefinitions": [
+          {
+              "name": "web",
+              "protocol": "tcp",
+              "port": 0
+          }
+      ],
+      "healthChecks": [
+          {
+              "portIndex": 0,
+              "path": "/",
+              "protocol": "HTTP"
+          }
+      ]
+  }
+  ```
+
+1. Deploy the `host-echo` service using the `host-echo.json` app definition by running the following command:
+
+  ```bash
+  dcos marathon app add examples/apps/host-echo.json
+  ```
+
+1. Open a text editor, then copy and paste the following Edge-LB pool configuration settings to define the `pool-http-with-aws-nlb` Edge-LB pool:
+
+  ```json
+  {
+    "apiVersion": "V2",
+    "name": "pool-http-with-aws-nlb",
+    "count": 1,
+    "cloudProvider": {
+      "aws": {
+        "elbs": [
+          {
+            "name": "echo",
+            "type": "NLB",
+            "internal": false,
+            "listeners": [
+              {
+                "port": 80,
+                "linkFrontend": "echo"
+              }
+            ],
+            "tags": [
+              {
+                "key": "Protocol",
+                "value": "HTTP"
+              }
+            ]
+          }
+        ]
+      }
+    },
+    "haproxy": {
+      "frontends": [
+        {
+          "name": "echo",
+          "bindPort": 80,
+          "protocol": "HTTP",
+          "linkBackend": {
+            "defaultBackend": "echo"
+          }
+        }
+      ],
+      "backends": [
+        {
+          "name": "echo",
+          "protocol": "HTTP",
+          "services": [
+            {
+              "marathon": {
+                "serviceID": "/host-echo"
+              },
+              "endpoint": {
+                "portName": "web"
+              }
+            }
+          ]
+        }
+      ]
+    }
+  }
+  ```
+
+1. Deploy the `pool-http-with-aws-nlb` pool configuration file to create the pool instance for load balancing access to the `host-echo` service by running the following command:
+
+  ```bash
+  dcos edgelb create examples/config/pool-http-with-aws-nlb.json
+  ```
+
+1. List the cloud provider load balancers managed by Edge-LB running in the attached DC/OS cluster by running the following command:
+
+  ```bash
+  dcos edgelb ingresslb pool-http-with-aws-nlb
+  ```
+
+  The command returns output similar to the following:
```

Comment:
As we introduced more detailed info about ingresslb, like EIP, the format of the CLI output has been also changed.
```
#1/1 loadbalancer
  NAME              DNS
  echo              dcos-lb-JOQ3wMJ7Z-Q8ayl-XzLP-PVv-f0f10cfccfa7d5a8.elb.us-west-2.amazonaws.com

  AVAILABILITY ZONE  ELASTIC IP

  LISTENER PROTOCOL  LISTENER PORT  FRONTEND
  TCP                80             echo
```
according to https://github.com/mesosphere/dcos-edge-lb/pull/415

Reply:
@mainred Lets update our internal docs with the above information as well.

Reply:
copied this output into doc.

Reply:
Hey @lsgunnlsgunn , I created a PR to update the doc in Edge-LB project https://github.com/mesosphere/dcos-edge-lb/pull/444. After Edge-LB PR is merged, we can then update this PR.